payload/prepare_request.go

```diff
@@ -32,35 +32,35 @@ type PrepareRequest interface {
 type prepareRequest struct {
 	transactionHashes []util.Uint256
 	nonce             uint64
-	timestamp         uint32
+	timestamp         uint64
```

Comment:
Again, change the interface, but don't change this `struct`, you can convert it in get/set methods.